tests/banks.py

```diff
@@ -108,6 +192,35 @@ def test_success_fetch_invalid_blocks(requests_mock):
     assert response == result
 
 
+def test_success_fetch_invalid_blocks_on_page_2(requests_mock):
+    results = [
+        {
+            "id": "2bcd53c5-19f9-4226-ab04-3dfb17c3a1fe",
+            "created_date": "2020-07-11T18:44:16.518695Z",
+            "modified_date": "2020-07-11T18:44:16.518719Z",
+            "block_identifier": "65ae26192dfb9ec41f88c6d582b374a9b42ab58833e",
+            "block": "3ff4ebb0-2b3d-429b-ba90-08133fcdee4e",
+            "confirmation_validator": "fcd2dce8-9e4f-4bf1-8dac-cdbaf64e5ce8",
+            "primary_validator": "51461a75-dd8d-4133-81f4-543a3b054149",
+        }
+    ]
+
+    address = "10.2.3.4"
+    url = f"http://{address}:80/invalid_blocks"
+
+    payload = {
+        "count": 3,
+        "next": None,
+        "previous": f"{url}?limit=2",
+        "results": results,
```

Comment:
This's true. Any reason to make it different than others.

Reply:
This is just a variation in results to not keep every fetch test the same.